Bottom: 60230df43fea47a917e663b9de20e7bfa04d6f90
Top:    e2d19e15bc9e75be8829fc649fe37d3beaedee23
Author: Jason Wang <jasowang@redhat.com>
Date:   2016-06-06 22:12:47 +0800

Refresh of vhost-fix-access-for

---

diff --git a/drivers/vhost/vhost.c b/drivers/vhost/vhost.c
index 7dd90f8..2024bd1 100644
--- a/drivers/vhost/vhost.c
+++ b/drivers/vhost/vhost.c
@@ -783,7 +783,7 @@ static int vhost_copy_from_user(struct vhost_virtqueue *vq, void *to,
 		struct iov_iter f;
 		ret = translate_desc(vq, (u64)from, size, vq->iotlb_iov,
 				     ARRAY_SIZE(vq->iotlb_iov),
-				     VHOST_ACCESS_WO);
+				     VHOST_ACCESS_RO);
 		if (ret < 0) {
 			vq_err(vq, "IOTLB translation failure: uaddr "
 			       "0x%llx size 0x%llx\n",
